update-server/Pipfile

```
@@ -3,12 +3,14 @@ url = "https://pypi.python.org/simple"
 verify_ssl = true
 name = "pypi"
 
+[requires]
+python_version = "3.7"
+
 [packages]
-"e1839a8" = {path = ".", editable = true}
-virtualenv = "*"
 aiohttp = "==3.4.4"
 
 [dev-packages]
+opentrons-update-server = {path = ".", editable = true}
```

Same story here. Distname is otupdate

```suggestion
otupdate = {path = ".", editable = true}
```